Logger.Tests/LogFactoryTests.cs

```diff
@@ -1,11 +1,37 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
+using System;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class LogFactoryTests
     {
-        
+
+        [TestMethod]
+        public void Test_ConfigureFileLogger_SetsFilePath()
+        {
+            LogFactory logFactory = new LogFactory();
+            FileLogger logger = (FileLogger)logFactory.CreateLogger("FileLogger");

```

Comment:
```suggestion
            BaseLogger logger = logFactory.CreateLogger(nameof(LogFactoryTests));
```
No need to cast to a file logger, you can create a new BaseLogger since FileLogger is a type of BaseLogger. Also, you shoud be using `nameof` here.